routers/api/v2/tokens.go

```diff
@@ -0,0 +1,64 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	v2 "github.com/unicsmcr/hs_auth/authorization/v2"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"go.uber.org/zap"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func (r *apiV2Router) ServiceToken(ctx *gin.Context) {
+	owner := ctx.PostForm("owner")
+	allowedURIs := ctx.PostForm("allowedURIs")
+	expiresAt := ctx.PostForm("expiresAt")
+
+	if len(owner) == 0 {
+		r.logger.Debug("service name was not provided in request")
+		models.SendAPIError(ctx, http.StatusBadRequest, "service name for the token must be provided")
```

Comment:
```suggestion
		models.SendAPIError(ctx, http.StatusBadRequest, "token owner must be provided")
```
Since the parameter is called `owner` not service name